src/components/menu/Menu.js

```diff
@@ -0,0 +1,100 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import { getDishById } from './helpers/menuHelper';
+import _omit from 'lodash/omit';
+
+//Constants
+import { INITIAL_STATE, CHECKOUT } from './constants/menu.general';
+
+//Components
+import MenuLinks from './components/menuLinks';
+import Dishes from './components/dishes';
+import Cart from './components/cart';
+
+//CSS
+import './menu.css';
+
+class Menu extends React.Component {
+  state = INITIAL_STATE;
+
+  resetCartItems = () => {
+    this.setState({
```

Comment:
can't you directly set it to INITIAL_STATE

Reply:
please set the state using INITIAL_STATE

Reply:
It's not possible Initial state is changing

Reply:
how can it change, it is defined in constants folder and by resetCartItems mean we need the state to be in initial state right?